src/plugins/search/test/search.e2e.js

```diff
@@ -347,66 +372,80 @@ describe('Search plugin', () => {
   describe('search result decorator', () => {
     it('should add default search result class to cells which mach the query', () => {
 
-      var hot = handsontable({
-        data: Handsontable.helper.createSpreadsheetData(5, 5),
+      const hot = handsontable({
+        data: Handsontable.helper.createSpreadsheetData(3, 3),
         search: true
       });
 
-      var searchResult = hot.search.query('2');
+      hot.getPlugin('search').query('2');
 
       render();
 
-      for (var rowIndex = 0, rowCount = countRows(); rowIndex < rowCount; rowIndex++) {
-        for (var colIndex = 0, colCount = countCols(); colIndex < colCount; colIndex++) {
-          var cell = getCell(rowIndex, colIndex);
-
-          if (rowIndex == 1) {
-            expect($(cell).hasClass(Handsontable.plugins.Search.DEFAULT_SEARCH_RESULT_CLASS)).toBe(true);
-          } else {
-            expect($(cell).hasClass(Handsontable.plugins.Search.DEFAULT_SEARCH_RESULT_CLASS)).toBe(false);
-          }
-        }
-      }
+      let cell = hot.getCell(0, 0);
+      expect($(cell).hasClass(hot.getPlugin('search').searchResultClass)).toBe(false);
```

Comment:
Please simplify this calls here and in next tests.

Reply:
Done.